internal/common/enums.rs

```diff
@@ -421,6 +421,16 @@ macro_rules! for_each_enums {
                 /// The ["alternate reverse" direction as defined in CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-direction#alternate-reverse).
                 AlternateReverse,
             }
+
+            // This enum describes the scrollbar visibility
```

Comment:
In order for this to be shown in the builtin enums docs, it needs to be a doc comment (with three `/`)

Also for the enum values.